torchtitan/components/validate.py

```diff
@@ -105,6 +113,13 @@ def validate(
             ):
                 break
 
+            # prepare attention mask
+            extra_kwargs : dict[str, Any] = {"attention_masks" : model_parts[0].get_attention_masks(
```

Comment:
For readability, can you make this into a function similar as: https://github.com/pytorch/torchtitan/blob/refs/heads/main/torchtitan/train.py#L416C9-L416C33